libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.scss

```
@@ -97,6 +97,7 @@ ion-title {
 
 ion-content {
   --background: transparent;
+  color: var(--kirby-modal-color);
```

Please use [the public API](https://ionicframework.com/docs/api/content#css-custom-properties):
```suggestion
  --color: var(--kirby-modal-color);
```